openfisca_core/taxbenefitsystems/tax_benefit_system.py

```diff
@@ -231,7 +236,7 @@ def load_extension(self, extension):
         """
         Loads an extension to the tax and benefit system.
 
-        :param string extension: The extension to load. Can be an absolute path pointing to an extension directory, or the name of an OpenFisca extension installed as a pip package.
+        :param str extension: The extension to load. Can be an absolute path pointing to an extension directory, or the name of an OpenFisca extension installed as a pip package.
```

Comment:
Shouldn't this be converted to google docstring style too ?

Reply:
Unfortunately I don't think there has been a clear decision made on the docstring style to be used. Missing this, I really just try to make the compiler happy so we can get the documentation to compile. Currently, there are no such style checks. I would prefer to defer unifying style to when we have a consensus on the style, and then use automatic formatters and add a linter on it.